courses/utils.py

```diff
@@ -67,18 +67,23 @@ def process_course_run_grade_certificate(course_run_grade):
     """
     user = course_run_grade.user
     course_run = course_run_grade.course_run
+
+    # A grade of 0.0 depicts a blacklist, consequently existing certificate
```

Comment:
Please change to:

`# A grade of 0.0 indicates that the certificate should be deleted`

Reply:
Updated